### PR TITLE
fix(portals-admin): Fix admin portal system-e2e

### PR DESCRIPTION
**Pull request by saevarma**
## What

Update system-e2e smoke test for admin portal.

## Why

Its broken.

## Screenshots / Gifs

N/A

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Comment by eirikurn** [comment]
Is this the problem? I was sure it was an issue with the test not supporting the prompt delegations flow as well as the user not having access to any admins on staging.

**Comment by saevarma** [comment]
> Is this the problem? I was sure it was an issue with the test not supporting the prompt delegations flow as well as the user not having access to any admins on staging.

The `idsLogin` utility handles the delegation selection [here](https://github.com/island-is/island.is/blob/eee88546bf043638db0acb173005932054522a49/apps/system-e2e/src/support/login.ts#L55)

But I will have another look at it as I needed to disable L58 locally for it to work locally, but the error in Tesults was reported after the login flow.

File: apps/system-e2e/src/support/login.ts [hunk 1]
```diff
@@ -55,7 +55,9 @@ export async function idsLogin(
   if (await page.url().startsWith(urls.authUrl)) {
     console.log('Still on auth site')
     const delegations = page.locator('button[name="SelectedNationalId"]')
-    await expect(delegations).toHaveCountGreaterThan(0)
+
+    expect(await delegations.count()).toBeGreaterThan(0)
```
[the remark this answers is not in the document]
Comment (saevarma) [reply apps/system-e2e/src/support/login.ts:59]: I´m looking closer into `toHaveCounterGreaterThan`. It is our custom addon but it is failing with `Error: expect: Property 'toHaveCountGreaterThan' not found.`
This change is a working workaround using native playwright api.

[the remark this answers is not in the document]
Comment (saevarma) [reply apps/system-e2e/src/support/login.ts]: This was unnecessary, it was just failing locally as playwright was not picking up the `playwright.configt.ts` file automatically. So undid the changes.

**Review by kristofer-lp8f — APPROVED** [review]
LGTH

**Review by eirikurn — APPROVED** [review]

**Pull request merged**